src/main/java/com/stripe/model/CreditNote.java

```diff
@@ -317,6 +318,32 @@ public static CreditNote create(CreditNoteCreateParams params, RequestOptions op
         ApiResource.RequestMethod.POST, url, params, CreditNote.class, options);
   }
 
+  /** Get a preview of a credit note without creating it. */
+  public static CreditNote preview(Map<String, Object> params) throws StripeException {
+    return preview(params, (RequestOptions) null);
+  }
+
+  /** Get a preview of a credit note without creating it. */
+  public static CreditNote preview(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url = String.format("%s%s", Stripe.getApiBase(), "/v1/credit_notes/preview");
+    return ApiResource.request(
+        ApiResource.RequestMethod.GET, url, params, CreditNote.class, options);
+  }
+
+  /** Get a preview of a credit note without creating it. */
+  public static CreditNote preview(CreditNotePreviewParams params) throws StripeException {
+    return preview(params, (RequestOptions) null);
+  }
+
+  /** Get a preview of a credit note without creating it. */
+  public static CreditNote preview(CreditNotePreviewParams params, RequestOptions options)
```

Comment:
Should all of those be named `previewCreditNote` since that's how we do it for void?

Reply:
we have gaveled not to rename